toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -864,3 +894,419 @@ func createLiveOSIsoImage(buildDir, baseConfigPath string, isoConfig *imagecusto
 
 	return nil
 }
+
+// extractIsoImageContents
+//
+//   - given an iso image, this function extracts its contents into the specified
+//     folder.
+//
+// inputs:
+//
+//   - 'buildDir':
+//     path build directory (can be shared with other tools).
+//   - 'isoImageFile'
+//     path to iso image file to extract its contents.
+//   - 'isoExpansionFolder'
+//     folder where the extracts contents will be copied to.
+//
+// outputs:
+//
+//   - creates a local folder with the same structure and contents as the provided
+//     iso image.
+func extractIsoImageContents(buildDir string, isoImageFile string, isoExpansionFolder string) (err error) {
+
+	mountDir, err := ioutil.TempDir(buildDir, "tmp-iso-mount-")
+	if err != nil {
+		return fmt.Errorf("failed to create temporary mount folder for iso:\n%w", err)
+	}
+	defer os.RemoveAll(mountDir)
+
+	mountParams := []string{isoImageFile, mountDir}
+	err = shell.ExecuteLive(false, "mount", mountParams...)
```

Comment:
Use `safemount.Mount` type. Unmounting safely and reliably is actually pretty difficult.

Reply:
pending.